src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final String DATE_PATTERN = "dd.MM.yyyy";
```

Comment:
The constant `DATE_PATTERN` should be private if it is not intended to be used outside of `SalaryInfo` class. This helps in encapsulating what is internal to the class.